hydromt/models/model.py

```diff
@@ -722,121 +636,6 @@ def get_tables_merged(self) -> pd.DataFrame:
             [df.assign(table_origin=name) for name, df in self.tables.items()], axis=0
         )
 
-    # model static maps
-    @property
-    def staticmaps(self):
```

Comment:
staticmaps has been removed now. In the PR can you make sure the linked issue will then get closed automatically?

Reply:
I've done so below!